### PR TITLE
feat: sms notification for usa carriers

### PR DESCRIPTION
**Pull request by thegutuproject**
### Testing

I moved the notification method into the failure section, disabled email notification and watched my phone for incoming SMS

### CODE REVIEW

**Comment by jef** [comment]
Cool work around. I like it!

File: README.md [hunk 1]
```diff
@@ -67,6 +67,9 @@ First, you're going to need to copy the `.env.example` to `.env`. The current op
 | `EMAIL_USERNAME` | Gmail address; e.g. `jensen.robbed.us@gmail.com` |
 | `EMAIL_PASSWORD` | Gmail password; see below if you have MFA |
 | `STORES` | List of [stores](#Supported-stores) you want to be scraped; optional, default: `nvidia` |
+| `PHONE_NUMBER` | 10 digit phone number. ONLY FOR USA. SMS CHARGES APPLY, for example: `123456789` |
```
Comment (jef) [inline README.md:70]: > 10 digit phone number

Uses 9 digit phone number as example 🤣 

Seriously note, look great!!!

File: README.md [hunk 2]
```diff
@@ -67,6 +67,9 @@ First, you're going to need to copy the `.env.example` to `.env`. The current op
...
+| `CARRIER` | Carrier you have. Only supports `sprint, tmobile, att, verizon` |
+| `NOTIFICATION_METHODS` | List of methods you'd like to be notified by, default: `email` |
```
Comment (jef) [inline README.md:72]: I'm thinking that we get rid of this environment and don't have this configuration. If the user sets the value, then they should assume that it's going to be sent.

**Review by jef — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (thegutuproject) [reply README.md:70]: 🤣  good catch! and thank you :)

File: README.md (hunk 2 above)
Comment (thegutuproject) [reply README.md]: I agree, i've removed it in this branch, should be ok now

**Review by jef — APPROVED** [review]

**Pull request merged**